modules/rostests/apitests/shell32/SHChangeNotify.cpp

```diff
@@ -288,6 +288,7 @@ static void DoStepCheck(INT iStage, INT iStep, LPCSTR checks)
         default:
         {
             assert(0);
+            lineno = 0;
```

Comment:
Maybe set it as the default value above at line 216 (like the `answer` pointer that is set to its default NULL value).